runtimes/common/src/fees.rs

```diff
@@ -78,6 +78,21 @@ where
 	}
 }
 
+pub struct ToAuthorCredit<R>(sp_std::marker::PhantomData<R>);
+impl<R> OnUnbalanced<CreditOf<R>> for ToAuthorCredit<R>
+where
+	R: pallet_balances::Config + pallet_authorship::Config,
+	<R as frame_system::Config>::AccountId: From<AccountId>,
+	<R as frame_system::Config>::AccountId: Into<AccountId>,
+	<R as pallet_balances::Config>::Balance: Into<u128>,
+{
+	fn on_nonzero_unbalanced(amount: CreditOf<R>) {
+		if let Some(author) = <pallet_authorship::Pallet<R>>::author() {
+			let _ = <pallet_balances::Pallet<R>>::resolve(&author, amount);
```

Comment:
Here also you can remove the leading `let _ =`

Reply:
Resolved: [f1efce4](https://github.com/KILTprotocol/kilt-node/pull/527/commits/f1efce41b8166eb4cf4c4b721ac73e332e846550)